src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Sdk.Razor.StaticAssets.ProjectSystem.targets

```diff
@@ -0,0 +1,27 @@
+<!--
+***********************************************************************************************
+Sdk.Razor.CurrentVersion.props
```

Comment:
Doesn't match the name of the file.

Reply:
Got rid of the file as we are not bringing in the typescript imports anymore and its not necessary